src/actions/WindowActions.js

```diff
@@ -869,30 +860,36 @@ export function patch(
   };
 }
 
-export function fireUpdateData(
-  entity,
-  windowType,
-  id,
+export function fireUpdateData({
+  windowId,
+  documentId,
   tabId,
   rowId,
   isModal,
-  isAdvanced
-) {
+  fetchAdvancedFields,
+  doNotFetchIncludedTabs,
+}) {
   return (dispatch) => {
-    getData(entity, windowType, id, tabId, rowId, null, null, isAdvanced).then(
-      (response) => {
-        dispatch(
-          mapDataToState(
-            response.data,
-            isModal,
-            rowId,
-            id,
-            windowType,
-            isAdvanced
-          )
-        );
-      }
-    );
+    getData({
+      entity: 'window',
+      docType: windowId,
+      docId: documentId,
+      tabId: tabId,
+      rowId: rowId,
+      fetchAdvancedFields: fetchAdvancedFields,
+      doNotFetchIncludedTabs: doNotFetchIncludedTabs,
+    }).then((response) => {
+      dispatch(
+        mapDataToState(
+          response.data,
+          isModal,
+          rowId,
+          documentId,
+          windowId,
+          fetchAdvancedFields
+        )
+      );
+    });
```

Comment:
catch ? does the BE responds every time with the desired content ?

Reply:
basically yes, if it does not fail :)

let's fix the catch part when it happens because now I don't know when it happens and how it shall be solved.